packages/mookme/src/loaders/hooks-resolver.ts

```diff
@@ -18,9 +18,10 @@ function matchExactPath(filePath: string, to_match: string): boolean {
  * A class defining several utilitaries used to load and prepare packages hooks to be executed
  */
 export class HooksResolver {
-  gitToolkit: GitToolkit;
-  root: string;
-  hookType: string;
+  private static readonly stagingHooks = ['pre-commit'];
+  private readonly gitToolkit: GitToolkit;
+  private readonly root: string;
+  private readonly hookType: string;
```

Comment:
Is it necessary to turn this into privates ? It is very java-ish and these property are used elsewhere (the `hookType` property for instance) ?

I'm not especially against this syntax (though I don't think private + getter/setters are necessary here) but I am a much bigger fan of consistency